tests/acceptance_test.go

```diff
@@ -1349,9 +1349,10 @@ func TestSingleNode(t *testing.T) {
 		}
 
 		if len(commitIds) != 3 {
-			t.Errorf("Expected 3 commit ids, got %d", len(commitIds))
+			t.Errorf("Expected 2 commit ids, got %d", len(commitIds))
```

Comment:
but it's looking for 3 `if len(commitIds) != 3 {` ?

Reply:
ta